src/Transport/Topology/Topologies/EndpointOrientedTopology.cs

```diff
@@ -90,6 +90,9 @@ private void InitializeContainer(SettingsHolder settings)
 
             var validationStrategyType = (Type)settings.Get(WellKnownConfigurationKeys.Topology.Addressing.Validation.Strategy);
             container.Register(validationStrategyType);
+
+            var publishersConfiguration = PublishersConfiguration.ConfigureWith(settings);
```

Comment:
:+1: 
I wrote the same thing on a previous comment, waiting for your replies https://github.com/Particular/NServiceBus.AzureServiceBus/pull/152#discussion_r56180854